src/helpers/index.tsx

```diff
@@ -20,18 +22,35 @@ export async function getMarketPrice({ networkID, provider }: { networkID: numbe
   return marketPrice;
 }
 
+export async function getTokenPrice() {
+  let tokenId = "olympus";
+  const resp = await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`);
+  let tokenPrice: number = resp.data[tokenId].usd;
+  return tokenPrice;
+}
+
 export function shorten(str: string) {
   if (str.length < 10) return str;
   return `${str.slice(0, 6)}...${str.slice(str.length - 4)}`;
 }
 
+export function formatCurrency(c: number, precision: number | undefined) {
+  if (precision === undefined) precision = 0;
+  return new Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+    maximumFractionDigits: precision,
+    minimumFractionDigits: precision,
+  }).format(c);
+}
+
 export function trim(number: number | undefined, precision: number | undefined) {
```

Comment:
Same thing with defaults here, makes things more readable, and you don't have to be defensive within the method.